### PR TITLE
Started load, adjusted extract

### PR DESCRIPTION
**Pull request by ieva7**
Load has global variables instead of hardcode now. 

Some load script functionality added

### CODE REVIEW

File: pipeline/load.py [hunk 1]
```diff
@@ -1,28 +1,53 @@
...
+    load_dotenv()
+    try:
+        return connect(host=environ["DB_HOST"],
+                   user=environ["DB_USER"],
+                   password=environ["DB_PASSWORD"],
+                   port=environ["DB_PORT"],
+                   dbname=environ["DB_NAME"])
+    except OperationalError:
+        print("Failed to connect to database.")
+        raise OperationalError()
```
Comment (jGreeman) [inline pipeline/load.py:22]: why catch the error to raise it again?

Comment (ieva7) [reply pipeline/load.py:23]: i needed to break the script somehow

Comment (ieva7) [reply pipeline/load.py:23]: very open to suggestions

File: pipeline/load.py [hunk 2]
```diff
@@ -1,28 +1,53 @@
...
+    except Exception as exc:
+        print(f"get_db_connection Error: {exc}")
 
 
-if __name__ == "__main__":
-    load_dotenv()
+def get_data_from_db(conn: connection, table: str, column = "*")-> pd.DataFrame:
+    """Connects to the database and returns dataframe of selected table and column(s)"""
+    with conn.cursor() as cur:
+        cur.execute(f"""SELECT {column} FROM {table};""")
```
Comment (jGreeman) [inline pipeline/load.py:31]: is this vulnerable to sql injection?

Comment (ieva7) [reply pipeline/load.py:31]: it's not consumer facing

File: pipeline/load.py (hunk 1 above)
Comment (jGreeman) [reply pipeline/load.py:23]: if you let it raise the error without catching it will it not break?

Comment (ieva7) [reply pipeline/load.py:23]: but I want to make the impression that I expected this error

Comment (ieva7) [reply pipeline/load.py]: fixed

File: pipeline/load.py (hunk 2 above)
Comment (jGreeman) [reply pipeline/load.py:28]: true, but good practice to do so

**Pull request merged**